docs/Refund.md

```diff
@@ -0,0 +1,29 @@
+# Refunds
+
+Refunds are one of the use cases that use the `Pull Payments` feature.
```

Comment:
imo it's better to say that refunds are built on top of the pull payments feature

Reply:
```suggestion
Refunds are one of the applications built on top of the `Pull Payments` feature.
```